### ISSUE TITLE
Wrong imported JAR

### ISSUE THREAD

**Issue by lppedd** [JaneWardSandy/eclipse-pde-partial-idea#19]
As you can see the `IStatus` interface points to the `com.ibm.websphere.v9.core` bundle instead of the `org.eclipse.equinox.common` bundle.

![image](https://user-images.githubusercontent.com/19871649/121665953-08bfbf80-caa9-11eb-93f1-1df81a1f9740.png)

![image](https://user-images.githubusercontent.com/19871649/121666628-c185fe80-caa9-11eb-8ad9-5cd50a7a6102.png)

My target is specified as:

![image](https://user-images.githubusercontent.com/19871649/121667162-496c0880-caaa-11eb-8160-7b6f67bafa8d.png)

**Comment by JaneWardSandy** [comment]
@lppedd Please attach `$moduleName.iml` file, I will check whether this is related to the library order in IDEA parsing.

**Comment by lppedd** [comment]
Hi @JaneWardSandy!
The `MANIFEST.MF` is

![image](https://user-images.githubusercontent.com/19871649/121854014-e967a300-ccf1-11eb-8921-127b9e238322.png)

```
Require-Bundle: org.eclipse.ui;bundle-version="3.4.1",
 org.eclipse.core.runtime;bundle-version="3.4.0",
 com.ibm.etools.iseries.rse.ui;bundle-version="7.5.2",
 com.ibm.etools.iseries.subsystems.qsys;bundle-version="7.5.2",
 org.eclipse.rse.ui;bundle-version="3.0.2",
 com.ibm.etools.iseries.toolbox;bundle-version="7.5.2",
 org.eclipse.rse.services;bundle-version="3.0.1",
 com.ibm.etools.iseries.services.qsys;bundle-version="7.5.0",
 org.eclipse.rse.core;bundle-version="4.6.3",
 org.eclipse.ui.console;bundle-version="3.6.201",
 org.eclipse.rse.subsystems.files.core;bundle-version="4.6.3"
```

And here is the module `.iml` file.
[iml.txt](https://github.com/JaneWardSandy/eclipse-pde-partial-idea/files/6646340/iml.txt)

**Comment by lppedd** [comment]
I see there are also many duplicated libraries. Mmmh no idea why!

**Comment by JaneWardSandy** [comment]
I see it was duplicated too, it project library invisible from model.getModuleLibraryTable(), so it was not remove in `cn.varsa.idea.pde.partial.plugin.helper.ModuleHelper#setLibrary(com.intellij.openapi.module.Module, com.intellij.openapi.roots.DependencyScope, boolean, java.util.Set<java.lang.String>, java.util.Set<java.lang.String>)`

**Comment by JaneWardSandy** [comment]
It seems related with order entries when resolve the same FQN class by IDEA.

<img width="722" alt="image" src="https://user-images.githubusercontent.com/15345355/121982233-ba206700-cdc1-11eb-91c4-157d1156bba7.png">
<img width="703" alt="image" src="https://user-images.githubusercontent.com/15345355/121982276-cc9aa080-cdc1-11eb-86cc-5559a0e8211a.png">
<img width="483" alt="image" src="https://user-images.githubusercontent.com/15345355/121982326-e3d98e00-cdc1-11eb-92da-f64591120396.png">

**Comment by lppedd** [comment]
@JaneWardSandy thanks! How do you intend to fix this?

**Comment by JaneWardSandy** [comment]
I think this can be fix in module library, call `rearrangeOrderEntries` and move required bundle and re-export bundle to first order, and sort by `Required-Bundle` order.

**Comment by JaneWardSandy** [comment]
@lppedd 
Rearrange order entries with `JDK`, `Module source`, `Bundle ClassPath`, `Required Bundle`, `Bundle from Re-exported`, `other project libraries`.
<img width="889" alt="image" src="https://user-images.githubusercontent.com/15345355/122024407-fc19cf00-cdfa-11eb-91da-3ce050bab214.png">

**Comment by lppedd** [comment]
@JaneWardSandy sounds good. Just a doubt: is the `Required-Bundle` ordering something that is specified in the standard or is it something we choose just because it makes things work?

**Comment by JaneWardSandy** [comment]
@lppedd I had test it in Eclipse, it seems it was standard resolve order.
And maybe I still need change some rule, Rearrange order entries with `JDK`, `Module source`, `Bundle ClassPath`, `Required Bundle and Bundle from Re-exported`, `other project libraries`.

Like figure 1, it was `org.eclipse.core.runtime`, `org.eclipse.osgi`, `org.eclipse.equinox.common`... `re-export`... and so on.

<img width="973" alt="image" src="https://user-images.githubusercontent.com/15345355/122143875-ebae3680-ce84-11eb-8eef-71e6801db8bb.png">

<img width="934" alt="image" src="https://user-images.githubusercontent.com/15345355/122143897-f7016200-ce84-11eb-852b-c9d83ea45a63.png">

**Comment by JaneWardSandy** [comment]
<img width="748" alt="image" src="https://user-images.githubusercontent.com/15345355/122150631-abed4c00-ce90-11eb-9dde-fb6bfcac0e5f.png">

**Comment by JaneWardSandy** [comment]
IR will fix and release in version 1.2.0
